opencsp/common/lib/csp/OpticOrientationAbstract.py

```diff
@@ -1,11 +1,11 @@
-from abc import abstractmethod, ABC
+from abc import abstractmethod
+import copy
 from warnings import warn
+
 from opencsp.common.lib.geometry.TransformXYZ import TransformXYZ
-from opencsp.common.lib.tool.typing_tools import strict_types
-import copy
 
 
-class OpticOrientationAbstract(ABC):
```

Comment:
Why remove the ABC (Abstract Base Class) inheritance? I'd have thought we would want that inheritance for any abstract class.

Reply:
Fixed in another PR it looks like